concurrency/3.0/_index.md

```diff
@@ -1,31 +1,46 @@
 ---
-title: "Jakarta Concurrency 3.0 (Under Development)"
-date: 2021-05-20
+title: "Jakarta Concurrency 3.0"
+date: 2022-02-28
 summary: "Jakarta EE 10 Release"
 ---
 Jakarta Concurrency provides a specification for using concurrency from application components without compromising container integrity while still preserving the Jakarta EE platform's fundamental benefits.
 
-# Plan
-Jakarta Concurrency 3.0 will target the Jakarta EE 10 platform release. 
+* [Jakarta Concurrency 3.0 Release Record](https://projects.eclipse.org/projects/ee4j.cu/releases/3.0.0)
+  * [Jakarta EE Platform 10 Release Plan](https://eclipse-ee4j.github.io/jakartaee-platform/jakartaee10/JakartaEE10ReleasePlan)
```

Comment:
Adding the link that you mentioned, but without removing either of the other links because it is unclear to me which was unnecessary and the 2.0 release review appears to have them both.